### PR TITLE
Feature: Add anywhere text when no location selected on PlacePicker

### LINKED ISSUE #766: Add anywhere option value into the SearchForm's place pickers
As a user, I can choose anywhere option from the SearchForm's PlacePicker.
If as a user, I don't select any location,  search location input field will be set to anywhere. 

ezgif com-optimize

It should look like this:
Screen Shot 2019-05-13 at 15 50 39

### PR DESCRIPTION
**Pull request by aalises**
Anywhere is placed instead on the PlacePickers when no place is selected. Due to limitations on `Tequila`, both places can't be empty at the same time but one of them can (either destination or origin), so a check that both aren't empty is performed on submit. Closes #766 

May-13-2019 17-25-33

### CODE REVIEW

**Comment by FilipMessa** [comment]
That state when origin and destination are equal is anywhere, is not very nice from UX. Probably we can do that like it is on kiwi.com, and require that origin must be chosen, and just destination can have value anywhere. 
What do you think? @aalises

**Comment by aalises** [comment]
@FilipMessa That was how it was done before, so just reverting back the condition on `SearchForm` would do it, shall I?

**Comment by FilipMessa** [comment]
@aalises Yes, but I would like to keep anywhere label for the destination location.

**Comment by aalises** [comment]
May-14-2019 10-59-48

File: packages/components/src/tripInput/TripInput.js [hunk 1]
```diff
@@ -22,6 +22,10 @@ type Props = {|
   +placeholder?: string,
 |};
 
+const getEmptyLocationPlaceholder = (field: string): string => {
```
Comment (FilipMessa) [inline packages/components/src/tripInput/TripInput.js]: What about `getAnywherePlaceholder`? Just an idea

Comment (aalises) [reply packages/components/src/tripInput/TripInput.js]: Would also be nice! but in the case of `From` it's not really anywhere... hahaha, we can change it later!

**Pull request merged**